### DISCUSSION TITLE
Sensitivity analysis doesn't work for all parameters

### DISCUSSION THREAD

**Discussion by Microbiotman** [Open-Systems-Pharmacology/Forum#1555] [Questions & Problems]
Hi all, 
I have a question about sensitivity analysis. Indeed, for a range between 6 and 7 for Lipophilicity, my sensitivity analysis doesn't give any result for each output but only <NaN>.
But when logP is <6 and > 7, it works !
Do you have any explanation ?

Thank you very much !

**Comment by Yuri05** [comment]
Can you attach your project?

**Reply by Microbiotman** [reply]
Thank you for your answer.
My project contains some delicate data... 
Is it possible to give informations you need without attach the project ?
Thank you for your help !

**Reply by Yuri05** [reply]
Hard to say. What do you mean by "my sensitivity analysis doesn't give any result for each output but only ."? Do you get NaNs in the sensitivity table for Lipophilicity?
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/bcb1f5fd-3d53-4d0f-b020-50467963b5fc)

**Reply by Microbiotman** [reply]
that's exactly what happens !

**Reply by Yuri05** [reply]
Does it also happen, if you try to calculate sensitivity for the Lipophilicity **only** (and for no other parameters)?

Can you then try to manually adjust Lipophilicity in your simulation and check if you get any NaNs in the PK values then?
E.g. if you use the default variation settings, run simulations with:
* `Initial_Lipophilicity/1.1`
* `Initial_Lipophilicity/1.05`
* `Initial_Lipophilicity*1.05`
* `Initial_Lipophilicity*1.1 `

If any of these simulations produces NaN in your PK parameter(s) of interest - that's the reason for NaNs in the sensitivity

![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/4ad95d87-0635-4672-90c9-539889b49145)

**Comment by wilbertdew** [comment]
Which PK-parameter are you looking at? Especially the AUCinf might return NaNs if the parameter values result in only the increase of a molecule concentration in a limited time-window, extrapolation to infinity is infinite. Similar for t1/2.
